### DISCUSSION TITLE
A public FauxPilot server sponsored by the community

### DISCUSSION THREAD

**Discussion by felipecrs** [moyix/fauxpilot#94] [Ideas]
Have you considered this option? I think some people would be very willing to sponsor a public instance of FauxPilot running somewhere where everyone can access for free.

**Comment by moyix** [comment]
Well, it's open source, so anyone is welcome to do this if they want! I'm happy to mention it in the README or on the Wiki. :) I don't have the bandwidth to set up and manage that though; I can barely keep up with maintenance on the project as it is.

**Comment by smart2pet** [comment]
+1

**Comment by smart2pet** [comment]
So I haven't got an eligible computer and is there somebody who wants to make a public server?

**Comment by smart2pet** [comment]
Moyix, would you let some people make a public server?

**Comment by moyix** [comment]
Yes, as I said earlier anyone is welcome to make a public server if they want, I just don't have the time / resources to run one myself.

**Comment by vkuehn** [comment]
It would be probably easier to set Up a public Server If GPU is not needed. Is that possible ?

**Comment by smart2pet** [comment]
Well, It's possible but it will lose a lot of performance.:(

**Comment by Chapoly1305** [comment]
I have given a try 16B with an A100, it takes 2 ~5 second to complete a 100 line max and 20 token inquiry. It feels Even if such service is available, there will be overwhelming of request to handle, which will make most of the people unhappy with the outcome. Don't even get started people may abuse and DoS. If anyone interested, a self hosted workstation with a dedicated graphic card is consider the optimal solution, renting a cloud service for this inside a company development group probably also considerable.

**Reply by felipecrs** [reply]
I agree.
